ML/MLClassification.py

```diff
@@ -379,4 +394,5 @@ def main(): #pylint: disable=too-many-statements
             del data
         del PromptDfSelForEff, FDDfSelForEff
 
-main()
+if __name__ == "__main__":
```

Comment:
Is this needed?

Reply:
If MLClassification.py is used as a script in some other file yes since the function main() would be executed